app/faucet.py

```diff
@@ -1,110 +1,141 @@
+    } for i in range(0, len(node_rpc_entires))]
+
+
+info_pairs = prepare_url_key_pairs(NODE_RPC, FAUCET_MNEMONIC)
+
 
 @bot.command(name='send', help='Send token from faucet')
 async def nine_nine(ctx, arg):
-    
+
     INVOCATION_COUNT.inc()
-    
-    if arg != None and len(arg) != 48:
+
+    if arg is not None and len(arg) != 48:
         # Inform of an invalid address
         await ctx.send(str(ctx.author.mention) + " That wallet address seems odd - sure you got that right?")
         return
-    
+
     if (str(ctx.channel.type) == "private"):
         # Forbid DM in discord
         await ctx.send("Hold on Capt'ain, you can't send me private messages !")
         return
-    
+
     else:
-        
         username = str(ctx.author.name)
-        
-        #ensure we comply with send frequency
+
+        # ensure we comply with send frequency
         r = redis.Redis(host=REDIS_IP, port=REDIS_PORT)
         if r.exists(username):
             await ctx.send(str(ctx.author.mention) + " You can only request once every [{}] second(s) !".format(ISSUE_INTERVAL))
             ISSUANCE_THROTTLED.inc()
             return
         else:
             r.set(name=username, value="set")
-            r.expire(username, ISSUE_INTERVAL)    
-        
-        
-        substrate = SubstrateInterface(
-            url=NODE_RPC
-        )
-        call = substrate.compose_call(
-        call_module='Balances',
-        call_function='transfer',
-        call_params={
-            'dest': arg,
-            'value': TOKENS_TO_SEND
-            }
-        )   
-        reply = ""
-        extrinsic = substrate.create_signed_extrinsic(call=call, keypair=keypair)
-        reply = substrate.submit_extrinsic(extrinsic, wait_for_inclusion=False)
-        
+            r.expire(username, ISSUE_INTERVAL)
+
+        extrinsic_url = []
+        for info in info_pairs:
+            rpc_url = info['rpc_url']
+            faucet_kp = info['faucet_kp']
+
+            with SubstrateInterface(url=rpc_url) as substrate:
+                call = substrate.compose_call(
+                    call_module='Balances',
+                    call_function='transfer',
+                    call_params={
+                        'dest': arg,
+                        'value': TOKENS_TO_SEND}
+                )
+                reply = ""
+                extrinsic = substrate.create_signed_extrinsic(call=call, keypair=faucet_kp)
+                reply = substrate.submit_extrinsic(extrinsic, wait_for_inclusion=False)
+                extrinsic_url.append(
+                    '{1} on {0}'.format(rpc_url, reply['extrinsic_hash']))
+
         # inform user
-        msg = " Awesome, you just received \"{}\" {}! The extrinsic hash is [{}]".format(TOKENS_TO_SEND/1000000000000000000, TOKEN_SYMBOL, reply['extrinsic_hash'])
+        msg = " Awesome, you just received {} Token! Please check these extrinsic hash. \n {}".format(
+            TOKENS_TO_SEND / 1000000000000000000,
```

Comment:
The figure (1000000000000000000) should be added to env or configurable file. 
It's also duplicated,

Reply:
Extract to env